test_aodndata/moorings/make_burst_test_file.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env python
+
+# Create test file for burst_average.py testing
+
+from collections import defaultdict
+from datetime import timedelta
+from numpy import ma
+
+import numpy as np
+
+from netCDF4 import Dataset, num2date
+
+from test_aodndata.moorings.test_burst_average import INPUT_FILE as TEST_FILE
+
+stats = defaultdict(list)
+
+
+def update_stats(burst, vals, qc):
+    """update stats for burst with given values and qc"""
+    v = ma.array(vals, mask=(qc>2))
+    stats['burst'].append(burst)
+    stats['TEMP'].append(v.mean())
+    stats['TEMP_burst_min'].append(v.min())
+    stats['TEMP_burst_max'].append(v.max())
+    stats['TEMP_burst_sd'].append(v.std())
+    stats['TEMP_num_obs'].append((~v.mask).sum())
+    # print('{b:2} {num:5} {mean:12} {sd:12}'.format(b=burst, num=stats['num'][-1], mean=stats['mean'][-1],
```

Comment:
but more importantly, I don't get this file. Where is "make_burst_test_file.py" being used ? Cannot find any ref. if it is to create a fake burst averaged file, I don't think this is its place

Reply:
Yes, I used it to edit some variable values in the test _input file_ (not averaged) for test_aodndata/moorings/test_burst_average.py, so that I could check a few of the values in the output, knowing exactly what they should be. It's only quick&dirty code that did the job once, but I figure it's worth keeping in here just in case we ever need to update that test file.

I can remove it from the repo if you still think it doesn't belong here.